cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,67 @@
-/// <reference types="cypress" />
-/// <reference types="../support" />
+import SettingPageObject from '../support/pages/setting.page.Object';
+import HomePageObject from '../support/pages/home.page.Object';
+import ProfilePageObject from '../support/pages/profile.page.Object';
+import SignInPageObject from '../support/pages/signIn.page.Object';
 
-describe('Settings page', () => {
-  before(() => {
+const settingPage = new SettingPageObject();
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
 
-  });
+describe('Setting page', () => {
+  let user;
 
   beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
+    cy.login();
+    cy.visit('/settings');
   });
 
   it('should provide an ability to update username', () => {
-
+    settingPage.typeUserName(user.username);
+    settingPage.clickUpdateButton();
+    settingPage.reloadPage();
+    homePage.assertHeaderContainUsername(user.username);
   });
 
   it('should provide an ability to update bio', () => {
-
+    settingPage.typeBio(user.username);
```

Comment:
It seems you are using `user.username` as the bio in the `typeBio` method. Ensure that this is intentional and that `user.username` is the correct data to be used for the bio field.